### PR TITLE
Replace topojson 

### LINKED ISSUE #5570: update topojson to topojson client, server, and simplify
### Summary

**What we're after:**
_We need to remove topojson as it has been deprecated. They recommend using the topojson client, server, and simplify packages directly _

https://www.npmjs.com/package/topojson

Screen Shot 2023-01-19 at 2 37 00 PM

### Related issues

_List any relevant related issue(s)_

- #[issue_number]
- #[issue_number]

### Completion criteria

_What does the end state look like? Include any behavior, appearance, etc required for this ticket to be 'complete.'_

### Tech steps or considerations (optional)

_List any considerations the tech team should know. Additionally, any specific tech steps can be included here._

- [ ] First
- [ ] Next
- [ ] Then

### Future work

_Other future work that may be needed following this issue's completion._

### PR DESCRIPTION
**Pull request by tmpayton**
## Summary (required)
- Resolves #5570 

This PR removes topojson and replaces it with topojson client. 

## Impacted areas of the application

General components of the application that this PR will affect:

-  maps 

## How to test
1. checkout this branch 
2. `npm i && npm run build`
3. `cd fec/`
4. `./manage.py runserver`
5. Check various maps to ensure they still work:
     - http://127.0.0.1:8000/data/candidates/president/presidential-map/
     - http://127.0.0.1:8000/data/elections/?cycle=2022&state=TX&district=19
     - http://127.0.0.1:8000/data/candidate/S0GA00559/?cycle=2020&election_full=true&tab=raising 
               - under individual contributions, group by state

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,6 +129,8 @@
     "sinon": "^1.17.2",
     "sinon-chai": "^2.8.0",
     "stringify": "^5.2.0",
+    "topojson-server": "^3.0.1",
+    "topojson-simplify": "^3.0.3",
```
Comment (patphongs) [inline package.json:133]: @tmpayton I believe we only need `topojson-client`. I don't believe we use these others. @johnnyporkchops @rfultz I don't know if you would be able to confirm that?

**Review by patphongs — COMMENTED** [review]
Thanks @tmpayton, I just had one comment for other front end engineers as to if we use any of the other scripts in the topojson library.

**Review by pkfec — COMMENTED** [review]
The maps render OK with the latest changes. Can you test and confirm if we need all these npm packages?

**Comment by tmpayton** [comment]
> The maps render OK with the latest changes. Can you test and confirm if we need all these npm packages?

Thank you for the review @pkfec! I will test that out.

**Comment by tmpayton** [comment]
@pkfec @patphongs The maps render without topojson simplify and server. I removed them from the PR!

**Comment by pkfec** [comment]
> @pkfec @patphongs The maps render without topojson simplify and server. I removed them from the PR!

@tmpayton Thanks for the update, changes look good.

**Review by pkfec — APPROVED** [review]

**Review by patphongs — APPROVED** [review]
Looks good @tmpayton, thanks for removing the extra libraries. Good to know we have them in  case we need them for future development work though.

**Pull request merged**